tests_integration/cli/config/rc-json/file.js

```diff
@@ -1,7 +1,7 @@
-function f() {
+function rcJson() {
   console.log.apply(null, [
-    "this file",
+    "rc-json/file.js",
     "should have trailing comma",
-    "and single quotes",
+    "and single quotes"
```

Comment:
Why was the trailing comma removed here.

Reply:
@azz I just thought I should check both configuration options which affect the output instead of one. Should I revert it back?

Reply:
Ah I see, that's fine.